.gitignore

```diff
@@ -12,6 +12,7 @@
 /tmp/*
 !/log/.keep
 !/tmp/.keep
+/db/*.sqlite3
```

Comment:
Looks like `db/development.sqlite3` and `db/test.sqlite3` files were added in this PR.